contents/handbook/growth/marketing/index.md

```diff
@@ -4,71 +4,17 @@ sidebar: Handbook
-
-- **Speed:** We want to be highly reactive, low process, and reliant on other teams as little as possible to ship things. We want to get stuff wrong quickly, then iterate.
-
-- **Helping PostHog users be successful:** Through our docs, tutorials, newsletter and beyond, we help PostHog users be more successful, both generally in their goals, but also more successful and powerful users of PostHog.
-
-- **Content for our ideal customer:** We deliver genuinely useful insights about things those customers care about (can be purely product-related, but also general advice). We're pitching our content at ‘self-servers’.
-
-- **Documentation** We work with product teams to maintain update-to-date, high quality docs. We work with Website & Docs to ensure users can discover what they want to know. 
- 
-### Things we want to be good at
-
-- **Social media:** Specifically Twitter, where we've seen good traction posting on James' personal account and the PostHog brand account. We do a minimal amount of posting on PostHog's LinkedIn account to keep it active, but it's not an important channel for us. We don't use any other social media channels. 
-
-- **Paid ads:** An agency should keep handling this for us, but we shouldn’t get better at it internally. This includes out of home, should we ever decide to do this. We see paid ads as a hygiene factor. 
-
-- **Developer influencers:** A small but growing trend in the industry, most of these are very expensive and overwhelmingly focused on teaching people how to code (i.e. not our target audience). We tried this with an agency and didn't get traction, but have seen some early promising signs going to developers directly. 
-
-### Things we might want to be good at but haven't tested yet
-
-- **Highly visible at industry events:** We could become known for giving good talks at events, because this enhances word of mouth and is a natural extension of being good at creating good content.
- 
-- **Being cool and interesting people in online communities:** We could do a lot more of this in relevant communities in Slack/Discord/Stack Overflow etc., and it's more engaging than one-way social media. We also increase awareness in relevant communities by sponsoring them. Overall, we want to be a net contributor to those communities.
-
-### Things we don't want to spend time on
+## How marketing works
 
-- **Big, highly coordinated marketing campaigns:** We can do them, but our reactive, short turnaround campaigns have been far more successful. 
+Marketing at PostHog is a collaborative effort dispersed across three teams:
 
-- **PR:** If we do word of mouth well, our community will be far more valuable/credible than an appearance in TechCrunch.
+- **Content & Docs:** Responsible for content marketing, SEO, and maintaining documentation.
+- **Words & Pictures:** Responsible for product marketing, art work, paid ads, and email marketing.
+- **Website & Vibes:** Responsible for website design, community, and backend.
```

Comment:
@joethreepwood @corywatilo @charlescook-ph Added this overview to the marketing handbook to clarify top-level ownership of stuff.